<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createDataReceiverDialogWindowFramework</class>
 <widget class="QDialog" name="createDataReceiverDialogWindowFramework">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Data Receiver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="createDataReceiversTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="createFileDataRecieverTab">
      <attribute name="title">
       <string>Receive Data From File</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="selectFileDataSourcePushButton">
         <property name="text">
          <string>Select Datasource File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>403</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="createFileDataReceiverPushButton">
         <property name="text">
          <string>Create File Data Receiver</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="filePathTextLabel">
         <property name="text">
          <string>FilePath</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createTCPDataReceiverTab">
      <attribute name="title">
       <string>Receive Data From TCP Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>227</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="TCPReceiverIPAddressLineEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="createTCPDataReceiver">
         <property name="text">
          <string>Create TCP Receiver</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source IP Address:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Source Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="TCPReceiverPortLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createZMQDataReceiver">
      <attribute name="title">
       <string>Receive Data from ZMQ Publisher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Source IP Address:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="ZMQDataReceiverIPAddressLineEdit"/>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QPushButton" name="createZMQDataReceiverLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Create ZMQ Receiver</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Source Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="ZMQDataReceiverPortLineEdit"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>createDataReceiversTabWidget</tabstop>
  <tabstop>selectFileDataSourcePushButton</tabstop>
  <tabstop>createFileDataReceiverPushButton</tabstop>
  <tabstop>TCPReceiverIPAddressLineEdit</tabstop>
  <tabstop>TCPReceiverPortLineEdit</tabstop>
  <tabstop>createTCPDataReceiver</tabstop>
  <tabstop>ZMQDataReceiverIPAddressLineEdit</tabstop>
  <tabstop>ZMQDataReceiverPortLineEdit</tabstop>
  <tabstop>createZMQDataReceiverLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
